testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -552,11 +554,51 @@ def test_ClusterCollection_get_centroids(self, cc):
         assert cc.get_centroids() == [1, 3, 5], \
                      "ClusterCollection centroids aren't as expected"
 
-    def test_Cluster_add_metadata(self, cluster):
+    def test_cluster_add_metadata(self, cluster):
         metadata = cluster.elements*10
         cluster.add_metadata('test', metadata)
         assert np.all(cluster.metadata['test'] == metadata), \
                      "Cluster metadata isn't as expected"
+        metadata = np.append(metadata, 9)
+        error_message = ("Size of metadata is not equal to the "
+                        "number of cluster elements")
```

Comment:
I think they are still off by a single white space, sorry. (note: you can just merge from these suggestions if you want, it's often a lot faster than going back to an editor to make the changes).

```suggestion
                         "number of cluster elements")
```

Reply:
I just fixed it now